website/static/js/keen.js

```diff
@@ -188,7 +189,7 @@ var KeenTracker = (function() {
                 payload.tech = {
                     browser: keenTracking.helpers.getBrowserProfile(),
                     ua: '${keen.user_agent}',
-                    ip: '${keen.ip}',
+                    ip: ip,
```

Comment:
It didn't look I could access the IP address here and then manipulate it, ${keen.ip} is a "dynamic IP address placeholder" from keen.